build.gradle

```diff
@@ -0,0 +1,33 @@
+task buildDockerImage(type:Exec) {
```

Comment:
For extra credit, you could use the docker plantir plugin. What's nice about is that it remembers that it built the image so you can make the buildDockerImage as a dependency and it will not "rerun" docker build (even if that is mostly a noop).

Reply:
This isn't being run automatically, so it wasn't a big deal.